ast/parser.go

```diff
@@ -618,10 +618,10 @@ func (p *Parser) parseRules() []*Rule {
 		return []*Rule{&rule}
 	}
 
-	if usesContains && !rule.Head.Reference.IsGround() {
-		p.error(p.s.Loc(), "multi-value rules need ground refs")
-		return nil
-	}
+	//if usesContains && !rule.Head.Reference.IsGround() {
+	//	p.error(p.s.Loc(), "multi-value rules need ground refs")
+	//	return nil
+	//}
```

Comment:
Should this be gated by the feature flag, too, in some way? 🤔

Reply:
Ah, I had completely forgotten I had commented this out 😄. Yes, I'll put this behind the flag 👍 .